charms/kfp-profile-controller/src/charm.py

```diff
@@ -237,7 +244,14 @@ def get_images(
                 if image_name in images:
                     images[image_name] = custom_image
                 else:
-                    self.log.warning(f"image_name {image_name} not in image list, ignoring.")
+                    logger.error(
+                        f"Image name `{image_name}` set in `custom_images` config not found in images list: {', '.join(images.keys())}."  # noqa E501
+                    )
```

Comment:
Instead of using noqa E501.
```suggestion
                    logger.error(
                        f"Image name `{image_name}` set in `custom_images` config not found in "
                        f"images list: {', '.join(images.keys())}."
                    )
```